sh/tmuxcomplete

```diff
@@ -27,7 +27,8 @@
 # Words including 2000 lines of history per pane
 #     sh tmuxcomplete -c '-S -2000'
 
-if ! tmux info &> /dev/null; then
```

Comment:
Oh you might be onto something. @wikimatze: Could you give that a try on clean v0.1.0?